currency/code.go

```diff
@@ -218,20 +212,13 @@ func (b *BaseCodes) Register(c string, newRole Role) Code {
 				}
 				stored[x].Role = newRole
 			}
-			if !stored[x].CaseSensitive {
-				format = true
-			}
-			return Code{Item: stored[x], upperCase: format}
+			return Code{Item: stored[x], upperCase: format || c == lower}
```

Comment:
Is there a use-case for when format is false, and c != lower ?
I would have thought not. Maybe I'm being stupid.

Reply:
Subject to this, I think we should be changing format if it *is* true, because we use this decision (`format || c == lower`) twice.
But I'm still skeptical about when it'd be exercised.
If we put that in a:
```
if !format && c != lower {
  format = true
}
```
then we can see if we can hit it or not with TestCoverage.

Reply:
I purged it completely a number will default to lower anyway.